json-lint/test/resources/schema/inventory/valid/adsense.test.json

```diff
@@ -23,7 +23,8 @@
       "ad-slot": "${SLOT}",
       "auto-format": "rspv",
       "width": "${WIDTH}",
-      "height": "${HEIGHT}"
+      "height": "${HEIGHT}",
+      "full-width": "true"
```

Comment:
we are trying to enforce native types here.. so we made the choice to enforce true (boolean) rather than "true" (string). 
Do "full-width": true

Reply:
Fixed